packages/main/test/pages/Menu.html

```diff
@@ -61,29 +61,30 @@
 
 	<ui5-button id="btnOpen2">Open Menu</ui5-button> <br/>
 	<ui5-menu id="menu2" header-text="My ui5-menu">
-		<ui5-menu-item text="New Folder" additional-text="Ctrl+F" icon="add-folder" disabled></ui5-menu-item>
+		<ui5-menu-item id="newFolder" text="New Folder" additional-text="Ctrl+F" icon="add-folder" disabled></ui5-menu-item>
 		<ui5-menu-item text="Open" icon="open-folder" starts-section>
 			<ui5-menu-item text="Open Locally" icon="open-folder" additional-text="Ctrl+K"></ui5-menu-item>
 			<ui5-menu-item text="Open from SAP Cloud" additional-text="Ctrl+L" disabled></ui5-menu-item>
 		</ui5-menu-item>
-		<ui5-menu-item text="Preferences" icon="action-settings" starts-section disabled></ui5-menu-item>
+		<ui5-menu-item id="preferences" text="Preferences" icon="action-settings" starts-section disabled></ui5-menu-item>
 		<ui5-menu-item text="Exit" icon="journey-arrive"></ui5-menu-item>
 	</ui5-menu>
 
-	<ui5-popover id="detailsPopover">
+	<ui5-responsive-popover id="detailsPopover" initialFocus="detailsLink">
```

Comment:
This looks strange on Mobile device - when the first item is disabled, on menu opens you get directly the dialog with content that describes why menu item is disabled, and you even don't see that the menu is opened...

Reply:
I've switched to a `ui5-popover` instance instead of `ui5-responsive-popover` as this example is not planned to work on mobile. It was done to test a specific desktop scenario for a stakeholder.